### PR TITLE
added more lighting to the world so color "pop" more

### PR DESCRIPTION
**Pull request by irwineffect**
Just added more light sources at various angles.

### CODE REVIEW

**Review by bkallaher — CHANGES_REQUESTED** [review]
Maybe add another light on the other side of the pool.

File: worlds/underwater.world [hunk 1]
```diff
@@ -17,9 +17,48 @@
           <shadows>1</shadows>
     </scene>
 
-    <light type="directional" name="sun">
-      <cast_shadows>true</cast_shadows>
-      <pose>100 0 100 0 1.52 0</pose>
+    <light type="directional" name="sun0">
+      <cast_shadows>1</cast_shadows>
```
Comment (skallaher) [inline worlds/underwater.world:21]: What's the difference here between 1 and true?

Comment (irwineffect) [reply worlds/underwater.world:21]: no difference as far as I could tell.

**Comment by irwineffect** [comment]
added one more light

**Review by bkallaher — APPROVED** [review]

**Comment by bkallaher** [comment]
Looks a lot better

**Comment by dcrossman** [comment]
Looks good!

**Pull request merged**